tests/scripts/local_monitoring_tests/rest_apis/test_namespace_reco_e2e_workflow.py

```diff
@@ -314,10 +317,12 @@ def test_list_recommendations_namespace_exps(cluster_type):
     list_reco_json = response.json()
 
     # Validate the json against the json schema
+    assert list_reco_json[0]['kubernetes_objects'][0]['namespaces']['recommendations']['data']
```

Comment:
@shekhar316 Shouldn't you be calling validate_local_monitoring_recommendation_data_present here as well instead of the assert?

Reply:
Yes, forgot to remove assert statement from here. 
Added `validate_local_monitoring_recommendation_data_present` function here.